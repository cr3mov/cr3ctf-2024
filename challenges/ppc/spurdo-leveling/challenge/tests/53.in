62080857024067490 995882916130066726
5 3
54 181
