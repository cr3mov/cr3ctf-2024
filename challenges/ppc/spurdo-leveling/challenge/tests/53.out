121048415069296207 62253470607066621 3458526144837035
